### PR TITLE
Create Test for the "pilet publish" cli command

### PR DESCRIPTION
**Pull request by int0x81**

### CODE REVIEW

File: tests/pilet.test.js [hunk 1]
```diff
@@ -39,4 +41,43 @@ describe("pilet", () => {
...
+        const port = 9000;
+        const testFeedUrl = `http://localhost:${port}/api/v1/pilet`;
+        const testFeedKey = "df133a512569cbc85f69788d1b7ff5a909f6bcfe1c9a2794283a2fc35175882c";
+
+        const localFeedService = spawn(`npm run sample-feed`, {
+            cwd: process.cwd(),
+            shell: true
+        });
+
+        await sleep(5000); //wait for feed service to come up
```
Comment (FlorianRappl) [inline tests/pilet.test.js:56]: This does not seem to be reliable. Can we instead just poll e.g. `http://localhost:${port}/api/v1/pilet` until we get 200 back?

File: tests/pilet.test.js [hunk 2]
```diff
@@ -39,4 +41,43 @@ describe("pilet", () => {
...
+        let publishOutput;
+
+        await publish.then(success => {
+            publishOutput = success.stdout.toString();
+        }).catch(err => {
+            publishOutput = err.stdout.toString();
+        });
+        
+        expect(publishOutput).not.toMatch(/\[[0-9]+\]\s+Failed/gm);
+
```
Comment (FlorianRappl) [inline tests/pilet.test.js:79]: OK can we check that the pilet is also really in the feed service, i.e., that `/api/v1/pilet` has 1 pilet? It should have had 0 before publishing.

File: tests/pilet.test.js [hunk 3]
```diff
@@ -1,11 +1,13 @@
 const path = require("path");
+const { spawn } = require("child_process");
+const { exec } = require('child_process');
```
[the remark this answers is not in the document]
Comment (int0x81) [reply tests/pilet.test.js]: resolved

File: tests/pilet.test.js (hunk 1 above)
Comment (int0x81) [reply tests/pilet.test.js]: I ran this multiple times and it always worked fine for me. I dont see the benefit of extra complexity here, but Nikita said he has a function for that in a similar case and wanted to take take of that.

File: tests/pilet.test.js (hunk 2 above)
Comment (int0x81) [reply tests/pilet.test.js:105]: added second `expect` statement that checks if the returned amount of items is 1

**Review by FlorianRappl — APPROVED** [review]

**Pull request merged**